### DISCUSSION TITLE
Allow Widget Shortcut Customization

### DISCUSSION THREAD

**Discussion by ImranR98** [breezy-weather/breezy-weather#1062] [Ideas]
### Describe your idea

Currently, tapping on the time in the widget asks the OS to open a clock app, and tapping on the date asks the OS to open a calendar app. However, a user may want to customize those shortcuts to open a different app that the OS does not recognize as a clock/calendar app. For example, Proton Calendar.

I hope this isn't considered a duplicate of issue #90 - my understanding/description of the problem in that issue was wrong. This proposed feature here would work more generally for any app a user might want to launch.

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.3 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Please contact Proton developers to let them know you would like to register their app as your default calendar app.

We don't plan on duplicating settings that already exist in Android natively ;)
